crates/core/tedge_agent/src/agent.rs

```diff
@@ -542,6 +543,7 @@ impl SmAgent {
         responses: &mut impl PubChannel,
         topic: &Topic,
     ) -> Result<(), AgentError> {
+        debug!("Persisting restart operation");
```

Comment:
remove debug statement.

Reply:
Why? It's a debug log and not `info` or `warn`. It can help with debugging in future.